### PR TITLE
Math magicians testing

### PR DESCRIPTION
**Pull request by mricanho**
For this Pull Request, I tested all the functionalities of my calculator, using Jest. I tested as well the rendering of the website with snapshots using react-test-renderer.

### CODE REVIEW

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Changes required❌

# Hello @mricanho 

#### You did a great job, the test covers the code extensively, however there are some issues we have to address to get it approved🤗.

* [x] Files not related to the project should not be committed (e.g. `.DS_Store`. .env) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- Never commit `.env` file to your github repository, this file is meant to contains sensitive information which could be stolen, add the path to your `.gitignore` so they never get committed.

* [x] The test fails when we divide a number by zero. Dividing a number zero should return `undefined`.

* [x] The test fails when `total` is a negative number and `next` is a negative number.

Check comments below

#### That is it kindly make the changes and  submit a re-review👨‍🔧😇 

#### Happy coding👨‍💻

Addo Forison

File: src/__test__/logic/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  });
+  it('Should sum two numbers', () => {
+    const result = Operate('2', '4', '+');
+    expect(result * 1).toBe(6);
+  });
+  it('Should sustrate two numbers value', () => {
+    const result = Operate('2', '4', '-');
+    expect(result * 1).toEqual(-2);
+  });
+  it('Should return a divided value', () => {
```
Comment (Forison) [inline src/__test__/logic/operate.test.js:19]: Failing division by zero
```
 it('Should sustrate two numbers value', () => {
    const result = Operate('2', '0', '/');
    expect(result * 1).toEqual(undefined);
  });
```

File: src/__test__/logic/calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    const value = (result.total * 1);
+    expect(typeof value).toEqual('number');
+  });
+  it('Should return null object', () => {
+    const result = Calculate(firstObject, 'AC');
+    expect(result).toStrictEqual({ next: null, operation: null, total: null });
+  });
+  it('Should return a decimal value', () => {
+    const result = Calculate(firstObject, '.');
+    expect(result).toStrictEqual({ next: null, operation: null, total: '0.' });
```
Comment (Forison) [inline src/__test__/logic/calculate.test.js:28]: * [x] This mutation is not happening in the main app, the calculator does not allow for users to enter decimals, kindly revisit the calculator and make the tested feature reflect.

**Review by uche-inyama — APPROVED** [review]
## Approved

### Hi Miguel :slightly_smiling_face: 

All the tests are now passing and .env was added to .gitignore file.

We are good here; go on with your next task.

#### Happy coding :man_technologist: 

#### ``` Uchechukwu Henry Inyama ```

**Pull request merged**